README.md

```diff
@@ -1011,6 +1012,13 @@ To export exists unified alerts to YAML use Export alerts
 ### Alerts troubleshooting 
 To troubleshoot alerts in clickhouse grafana plugin when enable `level=debug` in `log` section `grafana.ini` or via `GF_LOG_LEVEL=debug` environment variable.
 
+## Histogram support
+![Histogram](https://github.com/Altinity/clickhouse-grafana/assets/9621964/153dc5f9-44cd-444d-b966-34e64a28bd2c)
```

Comment:
it will not work when publish to Grafana
add image explicitly in `.github/images/` folder and `github.com/raw/...` url